packages/design-system/src/components/FormLabel/useFormLabel.tsx

```diff
@@ -78,7 +91,7 @@ export interface UseFormLabelProps extends FormFieldProps {
  *   - labelProps:   Props to be applied to a `FormLabel` (where we get the name)
  *   - fieldProps:   Props to be applied to the field (a.k.a., the input)
  *   - wrapperProps: Props to be applied to the wrapping element
- *   - bottomError:  A rendered React element representing a bottom-placed error
+ *   - errorElement:  A rendered React element representing a bottom-placed error
```

Comment:
the logic for `topError` might mean updating this description to say maybe:
> A rendered React element representing error placement

Reply:
Good catch